src/forms/FieldsetTime.jsx

```diff
@@ -0,0 +1,247 @@
+// @flow
+import * as React from 'react';
+import cx from 'classnames';
+import { LocalTime, ChronoField } from 'js-joda';
+
+import Flex from '../layout/Flex';
+import FlexItem from '../layout/FlexItem';
+import Select from './Select';
+
+export const HOURS_INPUT_CLASS = 'timeInput-hours';
+export const HOURS_INPUT_NAME = 'hour';
+export const MINUTES_INPUT_CLASS = 'timeInput-minutes';
+export const MINUTES_INPUT_NAME = 'minute';
+export const MERIDIAN_INPUT_CLASS = 'timeInput-meridian';
+
+type Props = React.ElementConfig<'input'> & {
+	value: string, // empty string or HH:mm
```

Comment:
Could you add Docgen comments for these props? You can see an [example in this PR](https://github.com/meetup/meetup-web-components/pull/598/files#diff-8c4ea546d2c1ed928f5f523260d0001cR19)